site/en/query/cquery.md

```diff
@@ -368,6 +368,12 @@ output groups as determined by the
 [`--output_groups`](/reference/command-line-reference#flag--output_groups) flag.
 It does include source files.
 
+All paths emitted by this output format are relative to the
+[execroot](https://bazel.build/remote/output-directories), which can be obtained
+via `bazel info execution_root`. Paths to files in the main repository also
+resolve relative to the project directory (assuming the `bazel-out` convenience
```

Comment:
Move parenthetical to prefix "if" clause?

Is "project directory" the right technical term, vs "workspace directory"?

Reply:
Otherwise, looks good.  Thanks!